README.md

```diff
@@ -159,17 +213,10 @@ exec($(lab completion xonsh))
 source <(lab completion zsh)
 ```
 
-## What about [GLab](https://github.com/profclems/glab)?
```

Comment:
hmmm, I'll leave the choice to y'all, but personally am a bit hesitant to fully remove this section. Mostly because it was done out of solidarity for each others tool. They added a similar section in their readme to callout lab https://github.com/profclems/glab#what-about-lab

Reply:
Oh cool! :)  I had assumed badly that it was out of date :)